Python/bltinmodule.c

```diff
@@ -422,6 +422,29 @@ builtin_callable(PyObject *module, PyObject *obj)
     return PyBool_FromLong((long)PyCallable_Check(obj));
 }
 
+static PyObject *
+builtin_breakpoint(PyObject *self, PyObject *args, PyObject *keywords)
+{
+    PyObject *hook = PySys_GetObject("breakpointhook");
+
+    if (hook == NULL) {
+        PyErr_SetString(PyExc_RuntimeError, "lost sys.breakpointhook");
+        return NULL;
+    }
+    Py_INCREF(hook);
+    PyObject *retval = PyObject_Call(hook, args, keywords);
```

Comment:
You might try to use the new METH_FASTCALL calling convention.

```
builtin_breakpoint(PyObject *self, PyObject **args, Py_ssize_t nargs, PyObject *kwnames) 
...
PyObject *retval = _PyObject_FastCallKeywords(hook, args, nargs, kwnames);
...
```

It prevents the creation a temporary tuple for 'args' (and dict for 'keywords') and so make your function faster :-D